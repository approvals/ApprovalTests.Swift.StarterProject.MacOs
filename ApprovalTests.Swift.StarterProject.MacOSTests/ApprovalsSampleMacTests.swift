@testable import ApprovalTests_Swift_StarterProject_MacOS
import ApprovalTests_Swift
import XCTest

class ApprovalsSampleMacTests: XCTestCase {

    func testList() throws {
        var names: [String] = ["Llewellyn", "James", "Dan", "Jason", "Katrina"]
        names.sort()
        try Approvals.verifyAll("", names);
    }
    
    func testText() throws {
        try Approvals.verify(UnitTests.getBenefits())
    }

}
