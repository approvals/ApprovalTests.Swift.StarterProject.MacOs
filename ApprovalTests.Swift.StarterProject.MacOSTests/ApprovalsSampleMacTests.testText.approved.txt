The four benefits to testing are:
1. specification
2. feedback
3. regression
4. granularity